pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -281,7 +281,7 @@ def test_reindex_dups(self):
         tm.assert_frame_equal(result, expected)
 
         # reindex fails
-        msg = "cannot reindex from a duplicate axis"
+        msg = "cannot reindex from a duplicate axis (this usually occurs when trying tp join / assign to a column when the index contains duplicate values)."
```

Comment:
you'll need to split this into multiple lines

Reply:
Also `tp` --> `to`